onnx/reference/ops/op_qlinear_conv.py

```diff
@@ -50,6 +50,19 @@ def _run(  # type: ignore
             X, W, B, auto_pad, dilations, group, kernel_shape, pads, strides
         ).astype(np.int32)
 
+        # w_scale could be a scalar or a 1-D tensor. A 1-D tensor means a per
+        # output channel quantization.
+        if np.size(w_scale) > 1:
+            if np.ndim(w_scale) != 1:
+                raise ValueError(
```

Comment:
Could you add tests for the uncovered lines? Thanks!

Reply:
Certainly!

I believe after https://github.com/onnx/onnx/pull/6460/commits/e5f702b338d3983c2cabdac2282f7ee8b57b482d all lines are covered. But possibly some of the coverage warnings are false positives? I am unsure how some of this is configured.

Is it possible that the coverage tool is running in an environment without ONNXRuntime, and therefore the main unit test for this op is being skipped because of this decorator?:
https://github.com/onnx/onnx/blob/1e3be88265ec2aa84a5177f8282d5d64bb6bb4a0/onnx/test/reference_evaluator_test.py#L1585
I am seeing even lines within the test itself being marked as not covered.

Reply:
In that test, onnxruntime is used to compute the expected values. Since the inputs are not random, we could hardcode them in the file.

Reply:
Attempted this in https://github.com/mcollinswisc/onnx/commit/40d39bff3fd286501514cecaa269e82a88e9eacf

Reply:
Coverage now shows as 100% for the patch:
https://github.com/onnx/onnx/pull/6460/checks?check_run_id=31813895659